BLaurenB/techtime
# Copy Images into assets/image & style flash message to green

#### What's this PR do?
Copies the images into the app/assets/images path.
Removes the 'images/' prefix from freelancers.csv
Adds a flash.scss file and imports it in the application.scss. This file has some very basic and generic styling to meet the green flash message styling from user story 5.

#### Where should the reviewer start- what are the pertinent changes and the sequence?
#### How should this be manually tested?
Add a user to the cart and then click the remove link. The message should be green for a successful removal.

Log in and see if a green message shows up at the top.

#### Any background context you want to provide?
I didn't delete the images from the public folder because I wasn't fully confident on why they were there (I know it was something about styling).

#### What are the relevant tickets?
Closes #39 
#### Screenshots (if appropriate)
<img width="927" alt="screen shot 2017-11-06 at 4 24 05 pm" src="https://user-images.githubusercontent.com/24700836/32470355-992cc914-c314-11e7-91c8-d09fbc175242.png">

#### Questions:
